ansible/playbook.yml

```diff
@@ -0,0 +1,58 @@
+- name: playbook
+  hosts: webservers
+  remote_user: ec2-user
+  tasks:
+  - name: Install git and python3
+    ansible.builtin.yum:
+      name: 
+        - git
+        - python3
+      state: latest
+    become: yes
+
+  - name: Install poetry
+    ansible.builtin.shell:
+      cmd: curl -sSL https://install.python-poetry.org | python3 -
+      creates: ~/.local/bin/poetry
+
+  - name: Create todoapp directory
+    ansible.builtin.file:
+      path: /opt/todoapp
+      state: directory
+      owner: ec2-user
+    become: yes
+
+  - name: Clone todo_app
+    ansible.builtin.git:
+      repo: 'https://github.com/8kristy/DevOps-Course-Starter.git'
+      dest: /opt/todoapp
+      version: exercise-4
+      force: yes
+
+  - name: Install poetry dependencies
+    ansible.builtin.shell:
+      cmd: rm poetry.lock; ~/.local/bin/poetry install
```

Comment:
Why are we removing `poetry.lock`? While this should "work", it means we are much less confident that the app is actually using the exact versions of dependencies that we used for testing!

Reply:
When I ran `poetry install` on the VM and the old `poetry.lock` file was there, I'd get an error (think it was something like "The lock file is not up to date with the latest changes in pyproject.toml.") and I couldn't run the application. From what I remember `poetry lock --no-update` didn't fix it either, so I found that just deleting the file and then doing the install works, although guess that's not the best option

I think this could be due to different poetry versions? I have 1.7.1 locally and the VM has 1.8.2

I couldn't find a good solution to this and didn't want to spend too much time on it but open to suggestions :)